src/config.rs

```diff
@@ -64,8 +60,7 @@ impl PbftConfig {
             peers: Vec::new(),
             block_duration: Duration::from_millis(200),
             message_timeout: Duration::from_millis(10),
-            commit_timeout: Duration::from_millis(4000),
-            idle_timeout: Duration::from_millis(30_000),
+            faulty_primary_timeout: Duration::from_millis(30_000),
```

Comment:
You could just use `Duration::from_secs`

Reply:
Updated